### PR TITLE
[CIAPP-5098] Add TeamCity integration to public docs

### PR DESCRIPTION
**Pull request by andrea-mosk**
### What does this PR do?

Add TeamCity integration to public docs

### Motivation

We are releasing TeamCity as a new integration

### Additional Notes

---

### Reviewer checklist
- [x] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [x] Check images for PII
- [x] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [inline content/en/continuous_integration/pipelines/teamcity.md:76]: How can I say "Reach out to us" without incurring in these warnings? Contact Datadog support?

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:27]: Same as above :/

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:31]: Mmm I prefer "provided through" here

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (AdrianLC) [reply content/en/continuous_integration/pipelines/teamcity.md:25]: I would wait for editorial review first. Otherwise you might have to update all of them twice

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:41]: Not sure I get your question, but the webhooks intake path is https://webhook-intake.{{DD_SITE}}/api/v2/webhook, where DD_SITE can be one of:

- datadoghq.com (default)
- datadoghq.eu
- us3.datadoghq.com
- us5.datadoghq.com

The Plugin just compose the path based on the DD_SITE provided in the parameter. Here, the value can be one of those basically.

Comment (AdrianLC) [reply content/en/continuous_integration/pipelines/teamcity.md:41]: ok that's great I was thinking the templating would give us only us1/us3/eu1 etc

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (alxch-) [inline content/en/continuous_integration/pipelines/teamcity.md:35]: Maybe there's a screenshot you can provide with exactly what people should input there?

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (alxch-) [inline content/en/continuous_integration/pipelines/teamcity.md:64]: I think we should give a recommendation on what people should do here, before we tell them what does not work

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:56]: Mmm I see that the only related thing that that page says is that the pipelines list won't display data if the default branch is incorrect, which is already implicitly said in the sentence, no?

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:35]: For the composite build? Unfortunately the dependencies they need to set really depend on how they structured their build chain :/ The only screenshot I can provide is how to create a composite build, but it's basically a "check" when creating a build, to define that it will be composite

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:64]: Makes sense, let me add that part here

Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:64]: Changed!

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (alxch-) [reply content/en/continuous_integration/pipelines/teamcity.md:35]: understood, I think this makes sense to add it anyway and maybe show what it looks like (a random pipeline with a composition build in the end). Sorry if it does not make sense, feel free to ignore

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (alxch-) [reply content/en/continuous_integration/pipelines/teamcity.md:56]: It also explains how to fix it

Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Fair, I've linked the term

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:35]: I've added a sample pipeline as an image, looks fine to me. WDYT?

**Comment by kassenq** [comment]
Nit - for consistency w/ other [provider documentation](https://docs.datadoghq.com/continuous_integration/pipelines/gitlab/?tab=gitlabcom) can we change, on the Queue time line, "View amount of time pipeline jobs sit in the queue before processing" to "View amount of time pipeline jobs wait in the queue before processing?" I don’t have a strong preference for either "sit" or "wait", just want it to be consistent. Otherwise LGTM

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (kassenq) [reply content/en/continuous_integration/pipelines/teamcity.md:25]: Ah just saw this comment - I think we should be consistent across providers, don't have strong preference for individual wording so will defer to you and/or docs team reviewer

**Comment by andrea-mosk** [comment]
> Nit - for consistency w/ other [provider documentation](https://docs.datadoghq.com/continuous_integration/pipelines/gitlab/?tab=gitlabcom) can we change, on the Queue time line, "View amount of time pipeline jobs sit in the queue before processing" to "View amount of time pipeline jobs wait in the queue before processing?" I don’t have a strong preference for either "sit" or "wait", just want it to be consistent. Otherwise LGTM

@kassenq I was sure that I copied the queue sentence from another provider, and indeed we use "sit" in Buildkite https://docs.datadoghq.com/continuous_integration/pipelines/buildkite/ 😂 . So I guess we are not consistent at the moment (although this is something negligible).

**Comment by kassenq** [comment]
> 

Ah okay! I wasn't aware. Let's merge these docs first and I can make a PR afterward to make the wording consistent

**Comment by maycmlee** [comment]
Created [DOCS-4700](https://datadoghq.atlassian.net/browse/DOCS-4700) for docs review.

**Review by kassenq — APPROVED** [review]

**Review by buraizu — CHANGES_REQUESTED** [review]
Hi team, thanks for the PR and the attention paid to the documentation here. I've left a review based on our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md), and also added my input to some of the active conversations.
Is it possible to change the screenshot so that it does not depict `master` as the default branch? This is due to our guidelines around [inclusive language](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#inclusive-language).
Thanks again, and let me know any questions or feedback.

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Done!

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Changed in this way! Only difference is that I used "build chain" instead of "buildchain".

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: It's fine to remove the committer part, changed

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Done!

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Changed

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Changed

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Thanks, changed

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Changed

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:76]: Changed, thanks!

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:81]: It's inside an html tag, so I needed to use <strong></strong> to make it bold, but done!

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:41]: Yeah, looks good

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:46]: Changed!

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:52]: Changed, thanks!

**Comment by andrea-mosk** [comment]
Thanks for the feedback, @buraizu! I have addressed all the feedback in the last 2 commits and I have also changed the screenshot to not show the `master` branch anymore (I used `test_branch` instead). Feel free to take a look at the changes :) I will take a look at the page preview later in the day or tomorrow morning! 

> Is Datadog support the correct team to answer this question, as it pertains to billing? Should this not be directed to a Customer Success Manager? If so, it looks like this is usually done by linking to mailto:success@datadoghq.com.

I checked this internally with the team, and they should reach out to the support actually

**Comment by andrea-mosk** [comment]
👍 Confirming that everything looks good on the preview from my side.

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (buraizu) [inline content/en/continuous_integration/pipelines/teamcity.md:62]: Would it make sense to move the troubleshooting section to the end of the doc, just before the `Further reading` section? This is the format used in most other integration docs.

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (buraizu) [inline content/en/continuous_integration/pipelines/teamcity.md:78]:
```suggestion
<div class="alert alert-danger"><strong>Note:</strong> The Git author email is used for
```

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (buraizu) [inline content/en/continuous_integration/pipelines/teamcity.md:81]:
```suggestion
(<strong>UserId</strong> or <strong>Author Name</strong>) are used. <a href="https://docs.datadoghq.com/help/" target="_blank">Reach out to the Datadog support team</a> if you have any questions about your use case.
```

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (buraizu) [reply content/en/continuous_integration/pipelines/teamcity.md:78]: Optional suggestion to update this to be red and more eye-catching because of the potential billing implications.

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (buraizu) [reply content/en/continuous_integration/pipelines/teamcity.md:81]: Thanks for confirming that support is the correct team here!

**Review by buraizu — APPROVED** [review]
Thanks for implementing those changes. I noticed a few minor issues and posted a question about moving the troubleshooting section, but otherwise this LGTM.

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md:78]: Makes sense to me, I'll double check that visually is still ok

File: content/en/continuous_integration/pipelines/teamcity.md
Comment (andrea-mosk) [reply content/en/continuous_integration/pipelines/teamcity.md]: Yeah makes sense! I've moved it

**Pull request merged**
